README.md

```diff
@@ -77,13 +77,24 @@ return [
 
 ## Usage
 
+``` php
+php artisan make:notification ExpoNotification
+```
+App\Notifications\ExpoNotification
 ``` php
 use NotificationChannels\ExpoPushNotifications\ExpoChannel;
 use NotificationChannels\ExpoPushNotifications\ExpoMessage;
 use Illuminate\Notifications\Notification;
 
-class AccountApproved extends Notification
+class ExpoNotification extends Notification
 {
+    private $message;
+    
+    public function __construct($message)
```

Comment:
The default notification created from the command php artisan make:notification does not come with the above constructor, I would prefer to remove the $message param from the constructor... in other words I prefer the to keep the old example. Thanks :)